docs-2.0/nebula-dashboard-ent/7.monitor-parameter.md

```diff
@@ -2,11 +2,13 @@
 
 This topic will describe the monitoring metrics in Nebula Dashboard.
 
-!!! note
+## Machine
 
-    The default unit in **Disk** and **Network** is byte. The unit will be changed with the data magnitude as the page displays. For example, when the flow is less than 1 KB/s, the unit will be Bytes/s.
+!!! note
 
-## Machine
+    - The operating system of the monitored machine is Linux by default.
```

Comment:
It does not make sense. The MONITORED MACHINE should not have a default operating system, because we don't know that. The default value, if there is one, should be `-` or simply a space ` `. Unless there is only one condition `Linux` because only Linux is supported.

Plz correct me if I misunderstand the situation.

Reply:
i think i need to reorganize this sentence ---->all the machine metrics listed below are for the Linux operating system.

Reply:
That makes sense lol